<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="Lakshmi" name="Wie" tests="1" failures="1" timestamp="7 Jul 2015 01:07:07 GMT" time="96.253" errors="0">
  <testcase name="testLeaplogin" time="91.327" classname="leaplogin.Browserstest">
    <failure type="java.lang.AssertionError" message="timeout">
      <![CDATA[java.lang.AssertionError: timeout
	at leaplogin.Browserstest.testLeaplogin(Browserstest.java:63)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 17 stack frames]]>
    </failure>
  </testcase> <!-- testLeaplogin -->
</testsuite> <!-- Wie -->
